apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -119,6 +119,13 @@ type MetricsConfigSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Create ServiceMonitors for Tempo components"
 	CreateServiceMonitors bool `json:"createServiceMonitors,omitempty"`
+
+	// EnableAlerts specifies if Alerts should be created for Tempo components.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Create ServiceMonitors for Tempo components"
+	EnableAlerts bool `json:"enableAlerts,omitempty"`
```

Comment:
Now we have `CreateServiceMonitors` and `EnableAlerts` fields, which is inconsistent.
Imho we should either rename `CreateServiceMonitors` -> `EnableServiceMonitors` or `EnableAlerts` -> `CreateAlerts`, wdyt?

Reply:
yep, sounds good